docs/rbe-setup.md

```diff
@@ -221,15 +221,37 @@ Some rules (like protobuf) are particularly sensitive to changes in environment
 
 [Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--incompatible_strict_action_env)
 
-### --action_env
+### --action_env, --test_env
 
-You can set environment variables that are available to actions with the `--action_env` flag. This is commonly used to set `BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN` which tells bazel not to auto-detect the C++ toolchain.
+You can set environment variables that are available to actions with the `--action_env` flag.
+If you want to set environment variables that are only available to test actions, you can use the `--test_env` flag.
+
+This is commonly used to set ennvironment variables such as `GO_TEST_WRAP_TESTV=1`, which modify how `go_test` rule handles the XML reports.
 
 ```bash
---action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
+--test_env=GO_TEST_WRAP_TESTV=1
 ```
 
-[Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--define)
+[`--action_env` Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--action_env)
+[`--test_env` Bazel docs](https://docs.bazel.build/versions/master/command-line-reference.html#flag--test_env)
+
+### --repo_env
+
+This flag set additional environment variables that are available to repository rules executions.
+It's commonly used to set environment variables that are used to augment how toolchains and external repositories are prepared before the build phase.
```

Comment:
Maybe drop `before the build phase` as Bazel no longer has cleanly separated phases.

Reply:
```suggestion
It's commonly used to set environment variables that are used to augment how toolchains and external repositories are prepared.
```